defect-tracker-server/src/main/java/com/defect/tracker/controller/SubModuleController.java

```diff
@@ -63,5 +64,18 @@
 
 	
 	
+	@PostMapping(value = EndpointURI.SUBMODULE_ADD)
+	public ResponseEntity<Object> createSubModule(@RequestBody SubModuleDto subModuleDto){
+		if(subModuleService.isSubModuleExistsByName(subModuleDto.getSubModuleName())) {
```

Comment:
check submodule name validation to same module id

Reply:
yes Progress ,

Reply:
I did It